db/migrate/20160927230136_enable_fk_message_digests_nodes.rb

```diff
@@ -0,0 +1,21 @@
+class EnableFkMessageDigestsNodes < ActiveRecord::Migration
+
+  class Bag < ActiveRecord::Base
+  end
+
+  class MessageDigest < ActiveRecord::Base
+    belongs_to :bag
+  end
+
+  def up
```

Comment:
A previous migration removes the foreign key.  The FK is the reason setting the field NOT NULL is not working, so it has to be removed and then added back.  Using a name for it makes this manipulation more reliable.